crates/wasi-http/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+    name        = "wasi-http"
+    version     = "0.0.1"
+    authors.workspace = true
+    edition.workspace = true
+    repository  = "https://github.com/WebAssembly/wasi-http"
+    license     = "Apache-2.0 WITH LLVM-exception"
+    description = "Experimental HTTP library for WebAssembly in Wasmtime"
+    readme      = "readme.md"
+
+[dependencies]
+    anyhow = { workspace = true }
+    bytes = "1.1.0"
+    hyper = { version = "1.0.0-rc.3", features = ["full"] }
+    tokio = { version = "1", features = ["full"] }
```

we don't need the full set of features
```suggestion
    tokio = { version = "1", default-features = false, features = ["net", "rt-multi-thread", "time"] }
```